de.dentrassi.pm.aspect.mvnosgi/src/de/dentrassi/pm/aspect/mvnosgi/internal/Constants.java

```diff
@@ -0,0 +1,10 @@
+package de.dentrassi.pm.aspect.mvnosgi.internal;
+
+public class Constants
+{
+    public static final String POM_FILE_NAME = "mvnosgi.pom";
```

Comment:
After some thoughts about it, perhaps it would be better to use a dynamic name that follows the normal pom file name but contains some additional marker (e.g. artifactid-version-mvnosgi.pom).